lib/worldwide/region.rb

```diff
@@ -435,13 +439,23 @@ def zip_autofill
 
     # is a postal code required for this region?
     def zip_required?
-      if zip_requirement.nil?
+      if @zip_requirement.nil?
```

Comment:
Use direct references to `@zip_requirement` variable to avoid circular referencing between the two methods.